gulp/bower.js

```diff
@@ -0,0 +1,84 @@
+var gulp = require('gulp');
+var plugins = require('gulp-load-plugins')();
+var sequence = plugins.sequence.use(gulp);
+
+var fs = require('fs');
+var del = require('del');
+var path = require('path');
+var deploy = require('gh-pages');
+
+var env = process.env;
+var pkg = require('../package.json');
+
+var out = {
+    bower: './out/bower',
+    repo: './out/bower-repo'
+};
+
+gulp.task('bower', sequence(
+    'clean-bower-folder',
+    ['copy-license', 'copy-bower-json', 'copy-bower-artifacts', 'copy-readme'],
+    'publish-bower-repo'
+));
+
+gulp.task('clean-bower-folder', [], function () {
+    return del([out.bower, out.repo]);
+});
+
+gulp.task('copy-license', [], function () {
+    return gulp.src('./license.md').pipe(gulp.dest(out.bower));
+});
+
+gulp.task('copy-bower-artifacts', [], function () {
+    var distFiles = './out/dist/**/*';
+    return gulp.src(distFiles).pipe(gulp.dest(out.bower));
+});
+
+gulp.task('copy-bower-json', [], function () {
+    var bowerJsonPath = './publish/bower.json';
+    return gulp.src(bowerJsonPath)
+        .pipe(plugins.template(pkg))
+        .pipe(gulp.dest(out.bower));
+});
+
+gulp.task('copy-readme', [], function () {
+    return gulp.src('./README.md')
+        .pipe(_addHeader('./publish/readme.header.md'))
+        .pipe(gulp.dest(out.bower));
+});
+
+gulp.task('publish-bower-repo', [], function (cb) {
```

Comment:
Well, got it! you can raise an issue in gulp-gh-pages, put a comment with a reference to the issue here to document, why you have chosen that solution and leave the implementation as is for now.